backend/probloom/prob/models.py

```diff
@@ -56,27 +56,45 @@ def info_dict(self):
             "userID": self.creator.user.id,
             "username": self.creator.user.username,
             "solved_num": solved_num,
-            "recommended_num": recommended_num
+            "recommended_num": recommended_num,
         }
 
     def solver_dict(self):
         sovler_set = self.solver.all()
         res = []
         for solver in sovler_set:
-            res.append({
-                "userID": solver.solved_user.user.id, 
-                "username": solver.solved_user.user.username
-            })
+            res.append(
+                {
+                    "userID": solver.solved_user.user.id,
+                    "username": solver.solved_user.user.username,
+                }
+            )
         return res
 
 
+class Choice(Model):
+    choice1 = TextField(max_length=1000, default="choice1")
+    choice2 = TextField(max_length=1000, default="choice2")
+    choice3 = TextField(max_length=1000, default="choice3")
+    choice4 = TextField(max_length=1000, default="choice4")
+
+
+class Problems(Model):
+    index = IntegerField(blank=True, null=True)
+    problem_type = CharField(max_length=100, default="default type")
+    statement = TextField(max_length=1000, default="default statement")
+    choice = ForeignKey(Choice, related_name="problems_choice", on_delete=CASCADE)
+    solution = CharField(max_length=10, default="default solution")
+    explanation = TextField(max_length=1000, default="default explanation")
```

Comment:
I think explanation have to be a separated model, and explanation field is redundant because Explanation model will have Foreign key to point ProblemSet Model.

Reply:
implemented on sprint 4